thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/alert/util/AnomalyReportGenerator.java

```diff
@@ -122,15 +141,21 @@ public void buildReport(long startTime, long endTime, List<MergedAnomalyResultDT
         String feedbackVal = getFeedback(
             anomaly.getFeedback() == null ? "Not Resolved" : "Resolved(" + anomaly.getFeedback().getFeedbackType().name() + ")");
 
+        DateTimeZone dateTimeZone = Utils.getDataTimeZone(anomaly.getCollection());
         AnomalyReportDTO anomalyReportDTO = new AnomalyReportDTO(String.valueOf(anomaly.getId()),
             getAnomalyURL(anomaly, configuration.getDashboardHost()),
-            String.valueOf(anomaly.getAvgBaselineVal()),
-            String.valueOf(anomaly.getAvgCurrentVal()),
-            anomaly.getDimensions().toString(),
-            String.format("%.2f", getTimeDiffInHours(anomaly.getStartTime(), anomaly.getEndTime())), // duration
+            String.format("%.2f", anomaly.getAvgBaselineVal()),
+            String.format("%.2f", anomaly.getAvgCurrentVal()),
+            getDimensionsString(anomaly.getDimensions()),
+            String.format("%.2f hours (%s to %s) %s",
+                getTimeDiffInHours(anomaly.getStartTime(), anomaly.getEndTime()),
```

Comment:
Why always in Hours? I think there is nice method in DateTime that gives duration in readable form